<launch>

    <node pkg="sbmpo_models" type="benchmark" name="benchmark" output="screen">
        <param name="runs" value="100"/>
        <param name="model_type" value="grid_2d"/>
    </node>

</launch>
